### PR TITLE
[Android] Convert ClientNotification class to Kotlin.

### PR DESCRIPTION
**Pull request by AenBleidd**
Cover ClientNotification class with unit tests.

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/client/ClientNotification.kt
Comment (AenBleidd) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/client/ClientNotification.kt]: FIxed

**Review by Isira-Seneviratne — APPROVED** [review]

**Comment by talregev** [comment]
I want to test it before merge. 
Please wait for my review. thank you.

**Review by talregev — APPROVED** [review]
I approve.
I compile and run it on oneplus one (android 10) and there was notification as it should,
also it crunch.

**Pull request merged**
